src/primitives.es6

```diff
@@ -33,7 +42,7 @@ export class ElementType {
   toRefract(options = {}) {
     const attributes = this.convertAttributesToRefract('toRefract');
     const initial = {
-      element: this.element,
+      element: this.elementName(),
```

Comment:
Idea: use getter/setter, so you can call `this.elementName` instead of `this.elementName()` since you have `this.meta`, `this.content` etc.